### DISCUSSION TITLE
train with GPU but with no CUDA (mac users)

### DISCUSSION THREAD

**Discussion by JINHXu** [explosion/spaCy#7957] [Help: Other Questions]
Hi,

This might be a dumb question, but I was training transformer models, which are recommended to train on GPU. Though I am using a mac, I think since I don't have an NVIDIA card I can't really install CUDA? Just wondering how everyone else deal with such situation. Help would be appreciated!

Cheers,
JX

**Comment by hp52** [comment]
You could try Google Colab.

**Reply by JINHXu** [reply]
That's an option! :D But I previously tried training a non-transformer model on colab, it was not great experience... I'd say it's much slower than training on my computer
